app/models.py

```diff
@@ -77,49 +83,75 @@ class Member(BaseModel):
     role: str
     status: str
 
+
 class CommentData(BaseModel):
     comment: str
 
+
 class Comment(CommentData):
     author: UUID4
     created_at: datetime
 
+
 class PostData(BaseModel):
     message: str
 
+
 class Post(PostData):
     id: UUID4
     author: UUID4
     created_at: datetime
     comments: List[Comment]
 
+
 class EventInput(BaseModel):
     title: str
     date: datetime
     address: str
     price: int
-    description: str # short info about the event
-    duration: Optional[int] # in hours
-    extraInformation: Optional[str] # more detailed practical information
+    description: str  # short info about the event
+    duration: Optional[int]  # in hours
+    extraInformation: Optional[str]  # more detailed practical information
     maxParticipants: Optional[int]
     romNumber: Optional[str]
     building: Optional[str]
     picturePath: Optional[str]
+    transport: bool
+    food: bool
+    active: bool
+
 
 class Event(EventInput):
     eid: UUID4
 
+
 class EventUpdate(BaseModel):
     title: Optional[str]
     date: Optional[datetime]
     address: Optional[str]
     description: Optional[str]
     maxParticipants: Optional[int]
+    active: Optional[bool]
+
+
+class Participant(BaseModel):
+    id: str
```

Comment:
a participant should follow the member model i.e id: UUID

Reply:
Works fine for non-seed participants as id is not interpreted as UUID. Cant figure the seed to adapt this :thinking:

Reply:
yes UUID are stored as strings, but we still want the model to only allow UUID as the member and participants model should be equal as they represent the same objects in the database. I had no problem running the seeding with UUID instead of string :thinking:

Reply:
Seeding is not the problem, but joining/leaving events with this config. I believe the objects are different from seed and event api inserts and conflicts or something.